src/tpm2_pytss/TCTILdr.py

```diff
@@ -67,3 +67,19 @@ def name_conf(self):
 
     def __str__(self):
         return self.name_conf
+
+    @classmethod
+    def is_available(cls, name=None):
```

Comment:
You can use `@stadicmethod` and drop `cls` as the method doesn't need it.
Also, please add type annotations and documentation for the method, even if the rest of class doesn't have it.